src/EnergyPlus/FluidProperties.hh

```diff
@@ -521,7 +529,8 @@ namespace FluidProperties {
                               int &GlycolIndex,           // Index to Glycol Properties
                               std::string_view CalledFrom // routine this function was called from (error messages)
     );
-
+#endif // GET_OUT
```

Comment:
Are you wanting to leave these `#ifdef`s ?

Reply:
I will probably take them out in a later PR that cleans up the caching. I can also take them out now.

Reply:
Let's defer to later.